awx/ui_next/src/components/FormField/sortErrorMessages.test.js

```diff
@@ -0,0 +1,81 @@
+import sortErrorMessages from './sortErrorMessages';
+
+describe('sortErrorMessages', () => {
+  let consoleError;
+  beforeEach(() => {
+    consoleError = global.console.error;
```

Comment:
I recommend a comment otherwise you need to look into the src to know why it's needed.